src/Life/Github.hs

```diff
@@ -80,6 +81,11 @@ cloneRepo (Owner owner) = do
         infoMessage "Using SSH to clone repo..."
         "git" ["clone", "git@github.com:" <> owner <> "/dotfiles.git"]
 
+-- | Create new branch with given branch name
+createNewBranch :: Branch -> IO ()
+createNewBranch (Branch branch) =
+    "git" ["checkout", "-b", branch]
```

Comment:
I wonder, if we should check if the branch already exist :thinking: cc @chshersh

Reply:
@vrom911 thanks for raising this point! This function will be used in multiple-machine workflow started in the issue here:

* https://github.com/kowainik/life-sync/issues/39

Unfortunately, I didn't have a clear vision of how this should work and I'm out of context at the moment... I propose to keep this function simple and patch later after we clarify the exact branch-based workflow. I assume this function would do a lot, like check whether the local branch exist, and if not, whether the remote branch exist (because we don't want to overwrite existing work) but it's hard for me at the moment to think about all possible cases.